### DISCUSSION TITLE
Sync stops after a few minutes.

### DISCUSSION THREAD

**Discussion by PSUHammer** [bpozdena/OneDriveGUI#149] [Q&A]
I have been successfully using the Onedrive utility for a year or so now and just found this GUI app.  Great work!  Makes life slightly easier for me.

What I am trying to do is sync all OneDrive media files stored in three specific folders (photos, videos, music) from my OneDrive environment, excluding all other content and folders.  I want a one-way sync/download to my local Ubuntu 22.04 Plex server.   This seems to have been working well.

I have installed this GUI client and appear to have set it up correctly.  I can start a sync but it pauses after a few minutes....almost as if it freezes and I have to re-start the sync to kickstart it again.  

Here are my sync options
![image](https://github.com/bpozdena/OneDriveGUI/assets/47612390/802b4b33-750c-43f8-a03f-aa141663bc67)

Here are my excluded folders
![image](https://github.com/bpozdena/OneDriveGUI/assets/47612390/4ef9505b-85a8-40fb-993f-ba816616d0b3)

In the logs I do notes SOME errors like:  A few files with mismatching hash or file size...not sure how that is working but it appears occasionally.

How is this handling things like renaming a file in OneDrive?  Does it then download and sync the rename?  What could cause the sync to pause when the baseline OneDrive service works as expected?

Would it make sense to kill all my local files and do a full re-sync of those three folders I am trying to keep in check?

I do see these errors before it bombs out....

`024-01-07 12:19:17,024 [OneDriveGUI.py:2221][fn=read_stderr][ERROR] - [USER] ERROR: File download size mis-match. Increase logging verbosity to determine why.
2024-01-07 12:19:17,024 [OneDriveGUI.py:2221][fn=read_stderr][ERROR] - [USER] ERROR: File download hash mis-match. Increase logging verbosity to determine why.
2024-01-07 12:19:17,024 [OneDriveGUI.py:2221][fn=read_stderr][ERROR] - [USER] INFO: Potentially add --disable-download-validation to work around this issue but downloaded data integrity cannot be guaranteed.
2024-01-07 12:19:17,024 [OneDriveGUI.py:2221][fn=read_stderr][ERROR] - [USER] ERROR: File download size mis-match. Increase logging verbosity to determine why.
2024-01-07 12:19:17,024 [OneDriveGUI.py:2221][fn=read_stderr][ERROR] - [USER] ERROR: File download hash mis-match. Increase logging verbosity to determine why.
`

** EDIT:   I just blew away all my local files and started a new sync/download for my three main folders.  So far, it appears that the client is not getting hung up or pausing.  Perhaps it had issues syncing file changes or edits with local copies?  Not sure.  Will see how it goes.

**Comment by bpozdena** [comment]
@PSUHammer it's great you worked around it. We will just not be able to find out what exactly the cause was. Perhaps a full resync would have fixed it too...

In any case I'll keep this issue open as I'd like to handle these errors better in the GUI . It will be done in the 1.1alpha branch, I'm just not sure when I'll get the time yet.
